src/docs/output-targets/custom-elements.md

```diff
@@ -75,20 +68,42 @@ export const config: Config = {
 | `auto-define-custom-elements` | A component and its children will be automatically defined with the `CustomElementRegistry` when the component's module is imported.                                                                                                                                                                                                                                                                      |
 | `single-export-module`        | All component and custom element definition helper functions will be exported from the `index.js` file in the output directory (see [Defining Exported Custom Elements](#defining-exported-custom-elements) for more information on this file's purpose). This file can be used as the root module when distributing your component library, see [below](#distributing-custom-elements) for more details. |
 
-> Note: At this time, components created not using JSX may not be automatically
-> defined. This is a known limitation of the API and users should be aware of
-> it
+> Note: At this time, components created not using JSX may not be automatically defined. This is a known limitation of the API and users should be aware of it.
+
+### dir
+
+_default: `'dist/components'`_
+
+This config option allows you to change the output directory where the compiled output for this output target will be written.
+
+### empty
+
+_default: `true`_
+
+If `true`, this config option will remove the contents of the [output directory](#dir) before writing the new build output.
 
 ### generateTypeDeclarations
 
+_default: `true`_
+
 By default, Stencil will generate type declaration files (`.d.ts` files) as a part of the `dist-custom-elements` output target through the `generateTypeDeclarations` field on the target options. Type declaration files will be placed in the `dist/types` directory in the root of a Stencil project. At this time, this output destination is not able to be configured.
 
 Setting this flag to `false` will not generate type declaration files for the `dist-custom-elements` output target.
 
-This flag defaults to `true` when omitted from a Stencil configuration file.
-
 > When set to generate type declarations, Stencil respects the export behavior selected via `customElementsExportBehavior` and generates type declarations specific to the content of that file.
 
+### inlineDynamicImports
+
+_default: `false`_
+
+Setting this flag to `true` will trigger inline dynamic imports for the under-the-hood Rollup bundle output. For more information, see [Rollup's documentation](https://rollupjs.org/guide/en/#outputinlinedynamicimports) for this option.
+
+### includeGlobalScripts
+
+_default: `false`_
+
+Setting this flag to `true` will include global scripts in the bundle and execute them once the bungle entry point in loaded.
+
```

Comment:
Omitted `minify` option here. I'll add that to my `externalRuntime` PR since it relates to some changes there

Reply:
Yeah! I'll cherry-pick over the commit from the other PR that I want to have for this to reference